src/clients/discord.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  AutocompleteInteraction,
+  ChatInputCommandInteraction,
+  Client,
+  Collection,
+  EmbedBuilder,
+  GatewayIntentBits,
+  Interaction,
+  Partials,
+  REST,
+  RESTPostAPIApplicationCommandsJSONBody,
+  Routes,
+  SlashCommandBuilder,
+} from "discord.js";
+
+export type Command = {
```

Comment:
ok, sam, this is pretty cool. i really like this general restructure.

Reply:
yay